Tweak.x

```diff
@@ -99,7 +102,20 @@ static AnimationStyle AnimationStyleForTableView(UITableView *tableView)
 						result.frame = original;
 					} completion:NULL];
 				});
-				break;
+			case AnimationStyleUh:
```

Comment:
This shouldn't say Uh. I initially didn't know how to refer to it. And clearly I don't pay attention.